sbpp_discord.sp

```diff
@@ -129,6 +134,22 @@ void SendReport(int iClient, int iTarget, const char[] sReason, int iType = Ban,
 	
 	json_object_set(jContent, "color", json_integer(GetEmbedColor(iType)));
 
+	char szURLBuffer[512];
+	GetConVarString(WebsiteBaseURL, szURLBuffer, sizeof(szURLBuffer));
+
+	if(!(StrEqual(szURLBuffer, "")))
+	{
+		json_object_set(jContent, "title", json_string("View on Sourcebans"));
+	
+		if(iType == Comms)
+			Format(sBuffer, sizeof sBuffer, "%s/index.php?p=commslist&searchText=%s", szURLBuffer, sTargetID);
+		else if (iType == Ban)
+			Format(sBuffer, sizeof sBuffer, "%s/index.php?p=banlist&searchText=%s", szURLBuffer, sTargetID);
+		else if (iType == Report)
+			Format(sBuffer, sizeof sBuffer, "%s/sourceban/index.php?p=admin&c=bans#^2", szURLBuffer);
```

Comment:
You sure this line is valid?
I suppose `/sourceban` leads to a missing page, no?

Reply:
You are correct, this does not work. I will fix it right away.